Cargo.toml

```diff
@@ -16,6 +16,7 @@ serde = { version = "1.0", default-features = false, features = ["derive"], opti
 serde-big-array = { version = "0.3.2", default-features = false, optional = true }
 
 [dev-dependencies]
+bincode = { version = "1.3.3"}
```

Comment:
The other dependencies set major/minor; the patch is set only if strictly required

Reply:
Also; unless we need to specify special dependencies directives, we should use the plain version instead of opening the attributes

Reply:
Done 👍